### ISSUE TITLE
Improving the "tell a friend" message at the end of the membership procedure

### ISSUE THREAD

**Issue by SunyataZero** [peoples-fund/peoples-fund#41]
Right now the standard message just says "Thought you might be interested in learning about and helping this organization. I think they do important work."

**Comment by frederikfederspiel** [comment]
Yeah... I know, it's not the most inspiring message... Isn't it fine for
now ? If not, please let me know what you would like to hear in a message
like that that would make you join us :).

2017-03-02 17:17 GMT+01:00 Tord <notifications@github.com>:
